usagov/vote-gov-drupal
# VOTE-583 remove browsersync from votegov theme

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-583

## Description

Remove browsersync from votegov theme

## Deployment and testing

### Post-deploy

1. cd into votegov theme and run `npm i`

### QA/Test

1. run `npm run dev` and make sure there are no errors
2. run `npm run build` and make sure there are no errors

## Checklist for the Developer




## Checklist for the Peer Reviewers
